stdlib/3/multiprocessing/__init__.pyi

```diff
@@ -1,12 +1,25 @@
 # Stubs for multiprocessing
 
-from typing import Any
+from typing import Any, Callable, Iterable, Mapping
 
 class Lock(): ...
-class Process(): ...
+class Process():
+    # TODO: set type of group to None
+    def __init__(self,
+                 group: Any = ...,
+                 target: Callable = ...,
+                 name: str = ...,
+                 args: Iterable[Any] = ...,
+                 kwargs: Mapping[Any, Any] = ...,
+                 daemon: bool = ...) -> None: ...
+
+    def start(self) -> None: ...
+
+    def terminate(self) -> None: ...
 
 class Queue():
-    def get(block: bool = ..., timeout: float = ...) -> Any: ...
+    def get(self, block: bool = ..., timeout: float = ...) -> Any: ...
+    def put(self, item: Any, block: bool = ..., timeout: float = ...) -> None: ...
 
 class Value():
     def __init__(typecode_or_type: str, *args: Any, lock: bool = ...) -> None: ...
```

Comment:
Oh man, looks like this one has the same bug (missing self). I don't think that ever worked. :-(